src/Routes/BaseController.ts

```diff
@@ -17,6 +17,9 @@ export class BaseController {
                 case ErrorCodes[ErrorCodes.InvalidParameterError]:
                     res.status(400).send(ex);
                     break;
+                case ErrorCodes[ErrorCodes.NotSupportedError]:
```

Comment:
Does this mean there's a feature that will eventually be supported but is currently just an empty stub?

Reply:
Yeah, I need to put in some work to support it, but it's much more than anticipated.  Putting this in for now till I get to it.